src/components/DropZone/DropZone.tsx

```diff
@@ -151,10 +151,10 @@ export class DropZone extends React.Component<CombinedProps, State> {
     return Object.keys(newState).length ? newState : null;
   }
 
-  private node: HTMLElement | null = null;
-  private dropNode: HTMLElement | HTMLDocument | null = null;
+  private node: React.RefObject<HTMLDivElement>;
```

Comment:
```suggestion
  private node: React.createRef<HTMLDivElement>();
```

Reply:
Will add these changes! 😄 

eg: `private node = React.createRef<HTMLDivElement>()` like this?